### ISSUE TITLE
Entity.Cpu.cpu_freq_current: has sporadically enormous decimal places

### ISSUE THREAD

**Issue by Baxxy13** [richibrics/IoTuring#19]
While testing is see that "cpu_freq_current" has often enormous decimal places...
Examples:
```
Entity.Cpu.cpu_freq_current: 2214.90675
Entity.Cpu.cpu_freq_current: 1447.7362500000002
Entity.Cpu.cpu_freq_current: 1234.55375
```

System is Proxmox (Debian) on Celeron J4125

**Comment by richibrics** [comment]
Fixed with [559dfc6](https://github.com/richibrics/IoTuring/commit/559dfc67502f9526d0c9d3394668afa16f2b6455)

**Comment by richibrics** [comment]
Correct in 10f35144e9b89f6ad2387718fe6ad0f10b28ac70
